components/authz-service/storage/postgres/migration/migration.go

```diff
@@ -106,33 +108,41 @@ func (c *Config) Migrate(dataMigConf datamigration.Config) error {
 		if err != nil {
 			return errors.Wrapf(err, "failed to set IAM v2 migration_status to %s", constants_v2.EnumInProgress)
 		}
+		l.Info("Setting up IAM data basics...")
 		err = legacy.MigrateToV2(ctx, db, migrateV1Policies)
 		if err != nil {
 			statusErr := legacy.RecordMigrationStatus(ctx, constants_v2.EnumFailed, db)
 			if statusErr != nil {
-				return errors.Wrapf(statusErr, "failed to set IAM v2 migration_status to %s:%s", constants_v2.EnumFailed, err.Error())
+				return errors.Wrapf(statusErr, "failed to set IAM migration_status to %s:%s", constants_v2.EnumFailed, err.Error())
 			}
-			return errors.Wrap(err, "IAM v2 force-upgrade failed")
+			return errors.Wrap(err, "IAM IAM data basics failed")
 		}
 		err = legacy.RecordMigrationStatus(ctx, constants_v2.EnumSuccessful, db)
 		if err != nil {
-			return errors.Wrapf(err, "failed to set IAM v2 migration_status to %s", constants_v2.EnumSuccessful)
+			return errors.Wrapf(err, "failed to set IAM migration_status to %s", constants_v2.EnumSuccessful)
 		}
 	}
 
 	// idempotent
+	l.Info("Checking for data migrations...")
 	err = dataMigConf.Migrate()
 	if err != nil {
 		return errors.Wrap(err, "IAM data migrations failed")
 	}
 
 	// perform remaining migrations
+	l.Infof("Checking for remaining schema migrations...")
 	err = m.Up()
 	if err != nil && err != migrate.ErrNoChange {
 		return errors.Wrap(err, "migrations failed")
 	}
 
-	l.Infof("Completed db migrations")
+	version, _, err = m.Version()
+	if err != nil {
+		return errors.Wrap(err, "failed to get version post-migration")
+	}
+
+	l.Infof("DB initialization complete at version %v", version)
```

Comment:
Thought it would be a bonus to actually show the final migration level in the log, too.